src/Core/Cache/DefaultCacheFactory.php

```diff
@@ -85,7 +86,8 @@ protected function isAPCUSupported()
     {
         static $apcuSupported = null;
         if (null === $apcuSupported) {
-            $apcuSupported = ApcuAdapter::isSupported();
+            // Need to check for CLI because Symfony won't: https://github.com/symfony/symfony/pull/25080
+            $apcuSupported = Director::is_cli() ? ini_get('apc.enable_cli') && ApcuAdapter::isSupported() : ApcuAdapter::isSupported();
```

Comment:
Can't/shouldn't the `ApcuAdapter::isSupported()` method do the `ini` check?

Reply:
That's the Symfony part

Reply:
We could make our own `ApcuAdapter` I guess, but that seems like over-engineering

Reply:
> We could make our own ApcuAdapter I guess, but that seems like over-engineering

for sure, don't do that :)